File: planit_poker_web/src/utils/dtos/SessionDto.ts
import { SessionStatus } from "../Enums";

export interface SessionDto {
  id: number;
  dateCreated: string;
  dateEnded: string;
  title: string;
  status: SessionStatus;
  customValues: string;
  valueType: string;
  creatorId: number;
  sessionManagerId: number;
}

File: backend/src/main/java/com/agiletools/estimo/dtos/SessionDto.java
package com.agiletools.estimo.dtos;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import java.time.LocalDateTime;


@Data
public class SessionDto {
    private Long id;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime dateCreated;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime dateEnded;
    private String title;
    private String status;
    private String customValues;
    private String valueType;
    private Long creatorId;
    private Long sessionManagerId;
}
